bottle_sqlite.py

```diff
@@ -102,7 +104,7 @@ def apply(self, callback, route):
 
         # Test if the original callback accepts a 'db' keyword.
         # Ignore it if it does not need a database handle.
-        argspec = inspect.getargspec(_callback)
+        argspec = inspect.getfullargspec(_callback)
```

Comment:
in python 3.3+ the new way to do this:

```python
argspec = inspect.signature(_callback)
        if keyword not in argspec.parameters:
            return callback
```

But your way provides better backwards compat.